msa-api/src/main/java/cz/muni/fi/pa165/msa/dto/RequestCreateDTO.java

```diff
@@ -6,21 +6,18 @@
 
 public class RequestCreateDTO {
 
-//    private UserDTO customer;
-
+    private UserDTO customer;
     private String location;
-
-//    private Set<MonsterDTO> heroes;
-
+    private Set<MonsterDTO> heroes;
```

Comment:
Heroes should should be of type Set HeroDTO. This is a mistake I made, but please fix this here and in RequestDTO too please.